<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterMainView</class>
 <widget class="QMainWindow" name="PlotterMainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>plotter++</string>
  </property>
  <property name="windowIcon">
   <iconset resource="plotter_resource.qrc">
    <normaloff>:/icons/resources/app_icon.png</normaloff>:/icons/resources/app_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tabGraphic">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="loGraphTab"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabValues">
       <attribute name="title">
        <string>Values</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="loValuesTab"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTextConsole">
       <attribute name="title">
        <string>Text console</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="loTextConsole"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabImages">
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="loImageTab"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPointCloud">
       <attribute name="title">
        <string>Point cloud</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="loScatterLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConfigure">
       <attribute name="icon">
        <iconset>
         <normaloff>:/root/resources/setting.png</normaloff>:/root/resources/setting.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Configure</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="loConfigureTab"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Load"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_As"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="plotter_resource.qrc">
     <normaloff>:/icons/resources/exit.png</normaloff>:/icons/resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="plotter_resource.qrc">
     <normaloff>:/icons/resources/about.png</normaloff>:/icons/resources/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About and Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="icon">
    <iconset resource="plotter_resource.qrc">
     <normaloff>:/icons/resources/save_as.png</normaloff>:/icons/resources/save_as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="icon">
    <iconset resource="plotter_resource.qrc">
     <normaloff>:/icons/resources/open.png</normaloff>:/icons/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="plotter_resource.qrc">
     <normaloff>:/icons/resources/save.png</normaloff>:/icons/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="plotter_resource.qrc"/>
 </resources>
 <connections/>
</ui>
